pages/common/terminalizer.md

```diff
@@ -0,0 +1,23 @@
+# terminalizer
+
+> Utility program which records your terminal and generate animated gifs or share a web player.
+
+- Create a global config directory:
+
+`terminalizer init`
+
+- Record your terminal and create a recording file:
+
+`terminalizer record {{file_name}}`
+
+- Play a recorded file on your terminal:
+
+`terminalizer play {{file_name}}`
+
+- Render a recording file as an animated gif image:
+
+`terminalizer render {{file_name}}`
+
+- Upload a recording file and get a link for an online player:
```

Comment:
Where does it upload ?

Reply:
Just mentioning the upload destination should be fine IMO.

Reply:
I simplified it...